PWGCF/EbyEFluctuations/Tasks/meanPtFlucId.cxx

```diff
@@ -455,50 +452,63 @@ struct MeanPtFlucId {
     return false;
   }
 
-  template <typename T>
-  PIDType identifyParticle(const T& track)
+  template <PIDType s1, PIDType s2, PIDType s3, typename T>
```

Comment:
Why is it needed to template the function with `s1`, `s2`, and `s3`? 
Everything is known at code writing time
Cannot them be passed as usual function arguments?